lib/Core/ExecutionState.cpp

```diff
@@ -101,33 +101,22 @@ ExecutionState::~ExecutionState() {
   while (!stack.empty()) popFrame();
 }
 
-ExecutionState::ExecutionState(const ExecutionState& state):
-    fnAliases(state.fnAliases),
-    pc(state.pc),
-    prevPC(state.prevPC),
-    stack(state.stack),
-    incomingBBIndex(state.incomingBBIndex),
-
-    addressSpace(state.addressSpace),
-    constraints(state.constraints),
-
-    queryCost(state.queryCost),
-    weight(state.weight),
-    depth(state.depth),
-
-    pathOS(state.pathOS),
-    symPathOS(state.symPathOS),
-
-    instsSinceCovNew(state.instsSinceCovNew),
-    coveredNew(state.coveredNew),
-    forkDisabled(state.forkDisabled),
-    coveredLines(state.coveredLines),
-    ptreeNode(state.ptreeNode),
-    symbolics(state.symbolics),
-    arrayNames(state.arrayNames),
-    openMergeStack(state.openMergeStack),
-    steppedInstructions(state.steppedInstructions)
-{
+ExecutionState::ExecutionState(const ExecutionState &state)
```

Comment:
Why? The previous way is way more readable.

Reply:
Agreed and fixed.